src/display/font_loader.js

```diff
@@ -121,6 +124,16 @@ class BaseFontLoader {
   }
 
   get isFontLoadingAPISupported() {
+    if (
+      typeof PDFJSDev === "undefined" ||
+      PDFJSDev.test("!PRODUCTION || TESTING")
+    ) {
+      return shadow(
+        this,
+        "isFontLoadingAPISupported",
+        !!this._document?.fonts && !this.styleElement
```

Comment:
Pull out the `!!this._document?.fonts` from here and below to a variable above so we don't have to keep these in sync.